site/docs/spec.md

```diff
@@ -483,25 +483,25 @@ Lists must use the [3-level representation](https://github.com/apache/parquet-fo
-| **`fixed(L)`**     | `binary`    | The length would not be checked by the ORC reader and should be checked by the adapter. |
-| **`binary`**       | `binary`    |                                                                                         |
-| **`struct`**       | `struct`    | ORC `uniontype` would also map to **`struct`**.                                         |
-| **`list`**         | `array`     |                                                                                         |
-| **`map`**          | `map`       |                                                                                         |
+| Type               | ORC type            | Notes                                                                                   |
+|--------------------|---------------------|-----------------------------------------------------------------------------------------|
+| **`boolean`**      | `boolean`           |                                                                                         |
+| **`int`**          | `int`               | ORC `tinyint` and `smallint` would also map to **`int`**.                               |
+| **`long`**         | `long`              |                                                                                         |
+| **`float`**        | `float`             |                                                                                         |
+| **`double`**       | `double`            |                                                                                         |
+| **`decimal(P,S)`** | `decimal`           |                                                                                         |
+| **`date`**         | `date`              |                                                                                         |
+| **`time`**         | `long`              | Stores microseconds from midnight.                                                      |
+| **`timestamp`**    | `timestamp`         |                                                                                         |
+| **`timestamptz`**  | `timestamp_instant` |                                       |
+| **`string`**       | `string`            | ORC `varchar` and `char` would also map to **`string`**.                                |
+| **`uuid`**         | `binary`            |                                                                                         |
+| **`fixed(L)`**     | `binary`            | The length would not be checked by the ORC reader and should be checked by the adapter. |
+| **`binary`**       | `binary`            |                                                                                         |
+| **`struct`**       | `struct`            | ORC `uniontype` would also map to **`struct`**.                                         |
```

Comment:
> ORC `uniontype` would also map to **`struct`**

Is this already supported in ORC?

Reply:
I don't think so, atleast I don't see any code handling this. Not sure why it is mentioned in the spec.

Reply:
Makes sense to remove it, @edgarRd @rdblue  fyi

Reply:
Done